appveyor.yml

```diff
@@ -0,0 +1,27 @@
+# Test against the latest version of this Node.js version
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "8"
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js
+  - ps: Install-Product node $env:nodejs_version
+
+  - npm install -g yarn
+
+  # install modules
+  - yarn install
```

Comment:
You may want `yarn install --frozen-lockfile` so that you know that what's in the lockfile is used exactly as it's listed.